core-feature-pack/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 # Location of wildfly
 if [ -z "$JBOSS_HOME" ]; then
-	JBOSS_HOME="/opt/wildfly"
+	JBOSS_HOME="/opt/$NAME"
```

Comment:
Note to self: I don't think this is directly part of WFCORE-1604 but it makes this script consistent with wildfly-init-redhat.sh which seems like a good thing.